### PR TITLE
Update documentation for Neon intrinsics

### PR DESCRIPTION
**Pull request by ngzhian**
Add a link to SIMDe's implementation status, and also fix emojis used in the table.

### CODE REVIEW

File: site/source/docs/porting/simd.rst [hunk 1]
```diff
@@ -1123,47 +1126,47 @@ For detailed information on each intrinsic function, refer to `NEON Intrinsics R
...
+   * - vand
+     - ✅ native
    * - vbic
      - ⚫ Not implemented, will trigger compiler error
    * - vbsl
-     - native
+     - ✅ native
    * - vcagt
-     - ⚠ Does not have direct implementation, but is emulated using fast NEON instructions
+     - ⚠️  Does not have direct implementation, but is emulated using fast NEON instructions
```
Comment (tlively) [inline site/source/docs/porting/simd.rst:1147]: Why have two spaces after this emoji but only one space after the check mark and X emojis?

**Review by tlively — APPROVED** [review]
LGTM with one small question!

File: site/source/docs/porting/simd.rst [hunk 2]
```diff
@@ -1123,47 +1126,47 @@ For detailed information on each intrinsic function, refer to `NEON Intrinsics R
...
+   * - vand
+     - ✅ native
    * - vbic
      - ⚫ Not implemented, will trigger compiler error
    * - vbsl
-     - native
+     - ✅ native
    * - vcagt
-     - ⚠ Does not have direct implementation, but is emulated using fast NEON instructions
+     - ⚠️ Does not have direct implementation, but is emulated using fast NEON instructions
```
Comment (tlively) [inline site/source/docs/porting/simd.rst:1147]: I don't see any visual difference between these lines. Is the emoji different but similar or something?

Comment (ngzhian) [reply site/source/docs/porting/simd.rst:1147]: at least on my browser, it was: Screen Shot 2021-09-22 at 8 37 04 AM

now it is: Screen Shot 2021-09-22 at 8 37 39 AM

Comment (ngzhian) [reply site/source/docs/porting/simd.rst:1147]: The bytes are different:

was: e2  9a a0 0a
now: e2 9a a0 ef b8 8f 

Think it has to do with the yellow fill?

**Pull request merged**